runtime/resolvers/sql.go

```diff
@@ -62,6 +64,14 @@ func newSQL(ctx context.Context, opts *runtime.ResolverOptions) (runtime.Resolve
 		return nil, err
 	}
 
+	// If there is a limit, convert it to an int
+	interactiveRowLimit := cfg.InteractiveSQLRowLimit
+	if props.Limit != "" {
+		interactiveRowLimit, err = strconv.ParseInt(props.Limit, 10, 64)
+		if err != nil {
+			return nil, err
+		}
+	}
 	olap, release, err := opts.Runtime.OLAP(ctx, opts.InstanceID, props.Connector)
```

Comment:
nit: logic above is unrelated to OLAP initialization, so add a newline

```suggestion
	}

	olap, release, err := opts.Runtime.OLAP(ctx, opts.InstanceID, props.Connector)
```